martin/src/pg/pool.rs

```diff
@@ -15,20 +15,26 @@ pub const POOL_SIZE_DEFAULT: usize = 20;
 
 /// We require `ST_TileEnvelope` that was added in [`PostGIS 3.0.0`](https://postgis.net/2019/10/PostGIS-3.0.0/)
 /// See <https://postgis.net/docs/ST_TileEnvelope.html>
-const MINIMUM_POSTGIS_VER: Version = Version::new(3, 0, 0);
-/// Minimum version of postgres required for [`MINIMUM_POSTGIS_VER`] according to the [Support Matrix](https://trac.osgeo.org/postgis/wiki/UsersWikiPostgreSQLPostGIS)
-const MINIMUM_POSTGRES_VER: Version = Version::new(11, 0, 0);
+const MINIMUM_POSTGIS_VERSION: Version = Version::new(3, 0, 0);
+/// Minimum version of postgres required for [`MINIMUM_POSTGIS_VERSION`] according to the [Support Matrix](https://trac.osgeo.org/postgis/wiki/UsersWikiPostgreSQLPostGIS)
+const MINIMUM_POSTGRES_VERSION: Version = Version::new(11, 0, 0);
 /// After this [`PostGIS`](https://postgis.net/) version we can use margin parameter in `ST_TileEnvelope`
-const RECOMMENDED_POSTGIS_VER: Version = Version::new(3, 1, 0);
-/// Minimum version of postgres required for [`RECOMMENDED_POSTGIS_VER`] according to the [Support Matrix](https://trac.osgeo.org/postgis/wiki/UsersWikiPostgreSQLPostGIS)
-const RECOMMENDED_POSTGRES_VER: Version = Version::new(12, 0, 0);
+const ST_TILE_ENVELOPE_POSTGIS_VERSION: Version = Version::new(3, 1, 0);
+/// Before this [`PostGIS`](https://postgis.net/) version, some geometry was missing in some cases.
+/// One example is lines not drawing at zoom level 0, but every other level for very long lines.
+const MISSING_GEOM_FIXED_POSTGIS_VERSION: Version = Version::new(3, 5, 0);
+/// Minimum version of postgres required for [`RECOMMENDED_POSTGIS_VERSION`] according to the [Support Matrix](https://trac.osgeo.org/postgis/wiki/UsersWikiPostgreSQLPostGIS)
+const RECOMMENDED_POSTGRES_VERSION: Version = Version::new(12, 0, 0);
 
 #[derive(Clone, Debug)]
 pub struct PgPool {
     id: String,
     pool: Pool,
-    // When true, we can use margin parameter in ST_TileEnvelope
-    margin: bool,
+    /// Indicates if `ST_TileEnvelope` supports the margin parameter.
+    ///
+    /// `True` if running postgis >= 3.1
```

Comment:
if keyword, should not be capitalized.

```suggestion
    /// `true` if running postgis >= 3.1
```